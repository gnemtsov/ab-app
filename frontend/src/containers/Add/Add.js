import React from 'react';
import Form from '../../components/UI/Form/Form';

import classes from './Add.css';

export default () => {
    return (
        <div
            className={classes.FormContainer}>
            <Form
                title="New department"
                api="/departments/test" />
        </div>
    );
};