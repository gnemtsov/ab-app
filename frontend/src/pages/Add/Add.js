import React from 'react';
import Form from '../../containers/Form/Form';

import classes from './Add.css';

export default () => {
    const conf = {
        buttonText: ['Add', 'Adding..'],
        doneText: 'Department was added!'
    }
    return (
        <div
            className={classes.FormContainer}>
            <Form
                conf={conf}
                title="New department"
                submitted={true}
                api="/departments/add" />
        </div>
    );
};