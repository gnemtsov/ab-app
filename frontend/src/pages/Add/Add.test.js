import React from 'react';

import { configure, render, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Add from './Add';

configure({ adapter: new Adapter() });

describe('Add', () => {
	it('Renders correctly', () => {
		const wrapper = shallow(<Add />);
		expect(wrapper).toMatchSnapshot();
	});
})
