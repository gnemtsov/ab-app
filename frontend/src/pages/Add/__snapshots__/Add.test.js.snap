// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Renders correctly 1`] = `
<div
  className="FormContainer"
>
  <Form
    api="/departments/add"
    conf={
      Object {
        "buttonText": Array [
          "Add",
          "Adding..",
        ],
        "doneText": "Department was added!",
      }
    }
    submitted={true}
    title="New department"
  />
</div>
`;
